### PR TITLE
Add messages to ledger connection process

### LINKED ISSUE #8168: Add descriptive messages to Ledger connection process
Currently connecting a Ledger is a cause of many user support requests. There are two points that we could improve a little bit that could probably alleviate 90%+ of the ledger-related email we get:

- [ ] Add a message "Looking for your Ledger" to the spinner during the connecting phase.
- [ ] When receiving an error.message from the connection that starts with the string `U2F`, Show a message "We had trouble connecting to your ledger, try reviewing [our hardware wallet connection guide](https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet) and try again."

### PR DESCRIPTION
**Pull request by PatrykLucka**
Currently connecting a Ledger is a cause of many user support requests. This PR:

- adds a message "Looking for your $1" to the spinner during the connecting phase.
- when receiving an error from the connection that contains the string U2F, shows a message "We had trouble connecting to your $1, try reviewing [our hardware wallet connection guide](https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet) and try again."
Where "$1" is device name.

Screenshot 2020-09-02 at 14 39 03

Fixes #8168

### CODE REVIEW

File: ui/app/pages/create-account/connect-hardware/index.js [hunk 1]
```diff
@@ -171,12 +174,28 @@ class ConnectHardwareForm extends Component {
...
+            href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="hw-connect__link"
+            style={{ marginLeft: '5px', marginRight: '5px' }}
+          >
+            {this.context.t('walletConnectionGuide')}
+          </a>
+          {this.context.t('andTryAgain')}
+        </p>
```
Comment (whymarrh) [inline ui/app/pages/create-account/connect-hardware/index.js:193]: We should be able to merge these into a single translatable message, passing in JSX for the substitutions where needed:

```jsx
<p className="hw-connect__error">
  {this.context.t('troubleConnectingToWallet', [this.state.device, (
    <a
      href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
      target="_blank"
      rel="noopener noreferrer"
      className="hw-connect__link"
      style={{ marginLeft: '5px', marginRight: '5px' }}
     >
       {this.context.t('walletConnectionGuide')}
     </a>
   )])}
</p>
```

With the combined message something like: `We had trouble connecting to your $1, try reviewing $2 and try again.`

File: ui/app/store/actions.js [hunk 2]
```diff
@@ -379,7 +379,8 @@ export function forgetDevice (deviceName) {
 export function connectHardware (deviceName, page, hdPath) {
   log.debug(`background.connectHardware`, deviceName, page, hdPath)
   return async (dispatch) => {
-    dispatch(showLoadingIndication())
+    const capitalizedDeviceName = deviceName ? deviceName[0].toUpperCase() + deviceName.slice(1) : ''
```
Comment (whymarrh) [inline ui/app/store/actions.js:382]: Can we use lodash's [`capitalize `](https://lodash.com/docs/#capitalize) here?

**Review by whymarrh — CHANGES_REQUESTED** [review]

File: ui/app/store/actions.js (hunk 2 above)
Comment (PatrykLucka) [reply ui/app/store/actions.js:382]: Good point, forgot we have lodash here 😅

**Review by whymarrh — APPROVED** [review]
LGTM, thanks!

**Pull request merged**